### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by yosefe**

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5730/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@MattBBaker can you pls post ORNL failure?

**Comment by MattBBaker** [comment]
```
[----------] 10 tests from test_rcache, where TypeParam = 
[ RUN      ] test_rcache.basic/mt_10
[       OK ] test_rcache.basic/mt_10 (11 ms)
[ RUN      ] test_rcache.merge_with_unwritable/mt_6
[       OK ] test_rcache.merge_with_unwritable/mt_6 (6 ms)
[ RUN      ] test_rcache.merge_merge_unwritable/mt_6
[       OK ] test_rcache.merge_merge_unwritable/mt_6 (6 ms)
[ RUN      ] test_rcache.merge_invalid_prot/mt_6
[       OK ] test_rcache.merge_invalid_prot/mt_6 (6 ms)
[ RUN      ] test_rcache.merge_expand_prot/mt_6
[       OK ] test_rcache.merge_expand_prot/mt_6 (5 ms)
[ RUN      ] test_rcache.merge_inv/mt_6
[       OK ] test_rcache.merge_inv/mt_6 (7 ms)
[ RUN      ] test_rcache.merge/mt_6
[       OK ] test_rcache.merge/mt_6 (24 ms)
[ RUN      ] test_rcache.shared_region/mt_6
[       OK ] test_rcache.shared_region/mt_6 (9 ms)
[ RUN      ] test_rcache.release_inuse/mt_6
[       OK ] test_rcache.release_inuse/mt_6 (20 ms)
[ RUN      ] test_rcache.get_unmapped/mt_6
/var/lib/jenkins/jobs/UCX/workspace/contrib/../test/gtest/ucs/test_rcache.cc:56: Failure
Error: Input/output error
terminate called after throwing an instance of 'ucs::test_abort_exception'
  what():  std::exception
make: *** [test] Aborted
```

**Comment by yosefe** [comment]
bot:ornl:retest

**Comment by yosefe** [comment]
bot:ornl:retest

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/5756/ for details (Mellanox internal link).

**Comment by yosefe** [comment]
@alinask pls take a look

File: src/uct/sm/knem/knem_md.c [hunk 1]
```diff
@@ -359,7 +359,7 @@ static ucs_status_t uct_knem_md_open(const char *md_name,
                 uct_knem_md_close(&knem_md->super);
                 return status;
             } else {
-                ucs_debug("Could not create registration cache for: %s",
+                ucs_debug("Could not create registration cache: %s",
```
Comment (alinask) [inline src/uct/sm/knem/knem_md.c]: could...

Comment (yosefe) [reply src/uct/sm/knem/knem_md.c]: i will merge it for now, maybe will make some script to fix all messages in the code..

**Pull request merged**